From b2b34771aa2b949e905858642a9176e2bbf89244 Mon Sep 17 00:00:00 2001
From: Devaraj K <devaraj@apache.org>
Date: Thu, 2 Jul 2015 10:20:31 +0530
Subject: [PATCH 1149/2748] YARN-3875. FSSchedulerNode#reserveResource()
 doesn't print Application Id properly in log.
 Contributed by Bibin A Chundatt.

(cherry picked from commit 37d7395773b5bb24aa522db38a2602df9a5ac184)
(cherry picked from commit e1ca886419833d1499c1793c2392eb59313a2d82)

Change-Id: I1aa96914bab2f58053d14448768639b24236db6b
---
 .../scheduler/fair/FSSchedulerNode.java            |   11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FSSchedulerNode.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FSSchedulerNode.java
index be08dff..c86201a 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FSSchedulerNode.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FSSchedulerNode.java
@@ -68,12 +68,13 @@ public synchronized void reserveResource(
             " on node " + this);
       }
 
-      LOG.info("Updated reserved container " + 
-          container.getContainer().getId() + " on node " + 
-          this + " for application " + application);
+      LOG.info("Updated reserved container " + container.getContainer().getId()
+          + " on node " + this + " for application "
+          + application.getApplicationId());
     } else {
-      LOG.info("Reserved container " + container.getContainer().getId() + 
-          " on node " + this + " for application " + application);
+      LOG.info("Reserved container " + container.getContainer().getId()
+          + " on node " + this + " for application "
+          + application.getApplicationId());
     }
     setReservedContainer(container);
     this.reservedAppSchedulable = (FSAppAttempt) application;
-- 
1.7.9.5

